internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json

```diff
@@ -5048,16 +5048,7 @@
         "Count_Person_0To4Years_Female_TwoOrMoreRaces",
         "Count_Person_0To4Years_Female_WhiteAlone",
         "Count_Person_0To4Years_Female_WhiteAloneNotHispanicOrLatino",
-        "Count_Person_0To4Years_HAAC_FEMALE",
```

Comment:
Expected deletes. SVs with unmapped race and ethnicity codes are corrected as part of US Census PEP imports refresh.